docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -46,7 +46,7 @@ Executors
 
 Individual jobs are executed on public CI services, currently:
 
-- Linux: TravisCI, CircleCI, Azure Pipelines
+- Linux: TravisCI, Azure Pipelines
 - Mac: TravisCI, Azure Pipelines
 - Windows: AppVeyor, Azure Pipelines
 
```

Comment:
Need to update the "Queue" subsection below for changed CI services as well.

Reply:
Thanks for pointing this out. I've updated the "Queue" section to no longer mention AppVeyor.

Reply:
I've also added a link and reference to GitHub Actions in the "Queue" section.

Reply:
Hmm, you forgot to add Github Actions to the entries above as suggested.